selectrum.el

```diff
@@ -278,6 +278,39 @@ This option is a workaround for 2 problems:
   wrapping."
   :type 'integer)
 
+(defcustom selectrum-candidate-transformations
+  '((match      :indicator "->"  :face success)
+    (truncation :indicator "..." :face shadow)
+    (newline    :indicator "\\n" :face warning)
+    (whitespace :indicator ".."  :face shadow))
+  "Indicators used to represent transformations in displayed candidates.
+This formatting does not affect the actual value of a candidate.
+By default, in the case of multi-line candidates, said candidates
+are flattened, long candidates are truncated, repeated whitespace
+is shortened, and the matching line in a multi-line candidate is
+displayed at the front.
+
+These values affect the behavior of `selectrum--ensure-single-lines',
+which does the formatting by replacing the found pattern (except
+for indicating a match, in which case the indicator is inserted
+between the matched line and the candidate).
```

Comment:
The docs of a user option better shouldn't refer to private variables and how it works internally.